### PR TITLE
[EWS] Rename 'patch' to 'change' in transfer-archive-to-s3

### PR DESCRIPTION
**Pull request by JonWBedard**
#### 7ac8d4e9e3c6df7ab609ae537d79bb6c6ad57001
<pre>
[EWS] Rename 'patch' to 'change' in transfer-archive-to-s3
https://bugs.webkit.org/show_bug.cgi?id=236706
<rdar://problem/89025868>

Reviewed by Aakash Jain.

* Tools/CISupport/Shared/transfer-archive-to-s3: Change --patch_id argument
to --change-id.
* Tools/CISupport/ews-build/steps.py:
(TransferToS3): Invoke transfer-archive-to-s3 with --change-id.
* Tools/CISupport/ews-build/steps_unittest.py:

Canonical link: https://commits.webkit.org/247373@main
git-svn-id: https://svn.webkit.org/repository/webkit/trunk@289987 268f45cc-cd09-0410-ab3c-d52691b4dbfc
</pre>

### CODE REVIEW

File: Tools/CISupport/Shared/transfer-archive-to-s3 [hunk 1]
```diff
@@ -29,8 +29,8 @@ def main():
     parser = argparse.ArgumentParser(add_help=True)
 
     group = parser.add_mutually_exclusive_group(required=True)
-    group.add_argument('--revision', action="store", help='Revision number or patch_id for the built archive')
-    group.add_argument('--patch_id', action="store", help='patch_id of the patch')
+    group.add_argument('--revision', action="store", help='Revision number or change identifier for the built archive')
+    group.add_argument('--change-id', dest='change_id', action="store", help='patch id or hash of specified change')
```
Comment (aj062) [inline Tools/CISupport/Shared/transfer-archive-to-s3]: do you want to use `change-id` or `change_id`

**Review by aj062 — APPROVED** [review]

File: Tools/CISupport/Shared/transfer-archive-to-s3 (hunk 1 above)
Comment (JonWBedard) [reply Tools/CISupport/Shared/transfer-archive-to-s3]: I think 'change-id' is more consistent with a command line option, even though the variable should clearly be 'change_id' in Python code. I did make the change deliberately.

**Comment by JonWBedard** [comment]
Landed https://commits.webkit.org/247372@main (7ac8d4e9e3c6)!

**Comment by JonWBedard** [comment]
> Landed https://commits.webkit.org/247372@main ([7ac8d4e](https://github.com/WebKit/WebKit/commit/7ac8d4e9e3c6df7ab609ae537d79bb6c6ad57001))!

It's actually https://commits.webkit.org/247373@main (f5066bb212c3655cfc6f0da3c3c765697cb9696c). Fun race condition in `git-webkit land` I'll look at tomorrow.

**Pull request merged**